src/main/java/pixy/PixyExceptions.java

```diff
@@ -0,0 +1,15 @@
+package pixy;
```

Comment:
perhaps creating a separate directory for exceptions might be better for possible code expansion in the future?